corehq/apps/sms/views.py

```diff
@@ -621,6 +622,17 @@ def get_mobile_worker_contact_info(domain_obj, user_ids):
 
 
 def get_contact_info(domain):
+    # If the data has been cached, just retrieve it from there
+    cache_key = 'sms-chat-contact-list-%s' % domain
+    cache_expiration = 30 * 60
+    try:
+        client = cache_core.get_redis_client()
+        cached_data = client.get(cache_key)
+        if cached_data:
+            return json.loads(cached_data)
+    except:
+        pass
```

Comment:
This should satisfy Stickler and work just as well...
```suggestion
    client = cache_core.get_redis_client()
    cached_data = client.get(cache_key)
    if cached_data:
        try:
            return json.loads(cached_data)
        except Exception:
            pass
```

Reply:
I had assumed the try was for the `get_redis_client` - if it's for the `json.loads`, I ought to be able to remove it altogether, I don't see how that could fail.

Reply:
Oh, interesting. In that case maybe the try/except should be around `get_redis_client` an catch `RedisClientError`?

https://github.com/dimagi/commcare-hq/blob/e5dc1bba4133946f47d32c62198febb59a864be4/corehq/ex-submodules/dimagi/utils/couch/cache/cache_core/__init__.py#L52-L53

Reply:
I did look around for Redis-related exceptions earlier and saw that, but I don't see anywhere else that we catch it, so I don't think it needs to be added here.